src/Form/Form.js

```diff
@@ -234,7 +235,10 @@ class Form {
     }
 
     autofillInput = (input, string, dataType) => {
-        setValue(input, string)
+        const successful = setValue(input, string)
```

Comment:
I started returning booleans from `setValue` to decide whether we should highlight the field or not. Especially useful for `select` elements, where we might not find an `option` to select.